README.md

```diff
@@ -81,13 +81,9 @@ cd server/docs/examples
 ./fetch_models.sh
 
 # Step 2: Launch triton from the NGC Triton container
-docker pull nvcr.io/nvidia/tritonserver:22.04-py3
-
 docker run --gpus=1 --rm -p8000:8000 -p8001:8001 -p8002:8002 -v/full/path/to/docs/examples/model_repository:/models nvcr.io/nvidia/tritonserver:22.04-py3 tritonserver --model-repository=/models
```

Comment:
nit: add a space after `-v` to make it more clear what the argument is: `-v/path/to/model_repo:/models` looks like one long token vs. `-v /path/to/model_repo:/models`

Reply:
Using `--net=host` instead of exposing specific port can be more concise

Reply:
@GuanLuo , can you provide the entire argument so I can use it correctly?

Reply:
docker run --gpus=1 --net=host -v/full/path/to/docs/examples/model_repository:/models nvcr.io/nvidia/tritonserver:22.04-py3 tritonserver --model-repository=/models